src/core/initializeComponentsFactory.js

```diff
@@ -22,35 +22,25 @@ const { ALLOY_COOKIE_NAME, ALLOY_COOKIE_TTL_IN_DAYS } = cookieDetails;
 
 const memoizedGetTopLevelDomain = memoize(getTopLevelCookieDomain);
 
-export default (
+export default ({
   componentCreators,
   logger,
-  createNamespacedStorage,
   createCookieProxy,
   createComponentNamespacedCookieJar,
-  createLifecycle,
-  createComponentRegistry,
+  lifecycle,
+  componentRegistry,
   createNetwork,
-  createOptIn
-) => config => {
-  const componentRegistry = createComponentRegistry();
-  const { imsOrgId, propertyId, cookieDomain } = config;
-  const cookieName = `${ALLOY_COOKIE_NAME}_${propertyId}`;
+  optIn
+}) => config => {
+  const network = createNetwork(config, logger, lifecycle);
+  const { imsOrgId, cookieDomain } = config;
+  const cookieName = `${ALLOY_COOKIE_NAME}_${imsOrgId}`;
```

Comment:
Nice! We will discuss this some more to decide if we need to support multiple property IDs inside the same cookie.